dart_flexible_sync/README.md

```diff
@@ -62,6 +62,8 @@ Then run this command:
 1. Create a new app following the instructions here: [Create an App with Atlas App Services UI](https://www.mongodb.com/docs/atlas/app-services/manage-apps/create/create-with-realm-ui).
     For the purpose of this sample you don't need to create an app from a template. You can just create an empty application.
 1. Go to the **Authentication** menu in the left panel and make sure the option "Allow users to log in anonymously" under `Authentication providers` tab is ON. Save and then click the button `Review draft & deploy`. Read [Authentication Providers](https://www.mongodb.com/docs/atlas/app-services/authentication/providers/) for more information about the other authentication types.
+1. Go to the **Rules** menu and select "Default roles and filters" under the service name. Choose `readAndWriteAll` and click the button `Add preset role`. Click the button `Save dtaft` and confirm. Then click the button `Review draft & deploy`.
```

Comment:
```suggestion
1. Go to the **Rules** menu and select "Default roles and filters" under the service name. Choose `readAndWriteAll` and click the button **Add preset role
**. Click the button **Save draft** and confirm. Then click the button **Review draft & deploy**.
```

suggestion applies to other instances of this line.

also note as a general best practice in technical documentation, UI elements names are **bolded** not `highlighted`.

Reply:
> also note as a general best practice in technical documentation, UI elements names are **bolded** not `highlighted`.

with this in mind, it looks like some other text in the readmes should be changed as well.